code/hud/hudtarget.h

```diff
@@ -26,6 +26,7 @@ struct weapon_info;
 #define MATCH_SPEED_THRESHOLD				0.1f		// minimum speed target must be moving for match speed to apply
 #define CARGO_RADIUS_DELTA					100		// distance added to radius required for cargo scanning
 #define CAPITAL_CARGO_RADIUS_DELTA		250		// distance added to radius required for cargo scanning
+static const int CARGO_RADIUS_REAL_DELTA = 50;
```

Comment:
The `static` here is not necessary since `const` already implies it. Also, maybe a comment describing what this is useful for would be nice here.